src/core_plugins/kibana/public/context/api/utils/sorting.js

```diff
@@ -1,5 +1,33 @@
 import _ from 'lodash';
 
+
+/**
+ * The list of field names that are allowed for sorting, but not included in
+ * index pattern fields.
+ *
+ * @constant
+ * @type {string[]}
+ */
+const META_FIELD_NAMES = ['_seq_no', '_doc', '_uid'];
+
+/**
+ * Returns a field from the intersection of the set of sortable fields in the
+ * given index pattern and a given set of candidate field names.
+ *
+ * @param {IndexPattern} indexPattern - The index pattern to search for
+ *     sortable fields
+ * @param {string[]} fields - The list of candidate field names
+ *
+ * @returns {string[]}
+ */
+function getFirstSortableField(indexPattern, fieldNames) {
+  const sortableFields = fieldNames.filter((fieldName) => (
+    META_FIELD_NAMES.includes(fieldName)
+    || (indexPattern.fields.byName[fieldName] || { sortable: false }).sortable
```

Comment:
If none of the configured fields match a meta field or index pattern field, context will throw a fatal error (as a result of sending a bad request to ES). Same thing happens if the field exists, but is not sortable. We should handle this more elegantly so the user knows how to fix the problem.

![screen shot 2017-06-02 at 11 46 07 am](https://cloud.githubusercontent.com/assets/6239176/26733591/4a650e9c-4789-11e7-8f3b-8f5c835f5d97.png)